Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,23 +1,136 @@
-import React from 'react';
-import './App.css';
-
-const App = () => {
-  return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
-    </div>
-  );
-};
+import React, { Component } from "react";
+import "./App.css";
+import CalculatorDisplay from "./components/DisplayComponents/CalculatorDisplay";
+import ActionButton from "./components/ButtonComponents/ActionButton";
+import NumberButton from "./components/ButtonComponents/NumberButton";
+
+class App extends Component {
+  state = {
+    total: ""
+  };
+
+  handledisplayChange = value => {
+    this.setState({
+      total: (this.state.total += value)
+    });
+    console.log(this.state.total);
+  };
+
+  handleClear = () => {
+    this.setState({
+      total: (this.state.total = "")
+    });
+  };
+
+  handleEquals = () => {
+    this.setState({
+      total: eval(this.state.total)
+    });
+  };
+
+  render() {
+    return (
+      <div className="main-container">
+        <div className="display">
```

Comment:
I would add more components here to clean up this view. I can see that you laid it out as it would render but it makes it difficult to read.